http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/WsmApiClientProvider.scala

```diff
@@ -74,7 +72,6 @@ class HttpWsmClientProvider[F[_]](baseWorkspaceManagerUrl: Uri)(implicit F: Asyn
           super.performAdditionalClientConfiguration(clientConfig)
           ctx.span.foreach { span =>
             clientConfig.register(new WithSpanFilter(span))
-            clientConfig.register(new JerseyTracingFilter(Tracing.getTracer))
```

Comment:
I had to remove this to not enter dependency hell, and the `WithSpanFilter` is the only one that has docs about what it is actually providing us in terms of logging.

If we bump the wsm library it conflicts with the ridiculous amount of excludes we have and no longer works. 

We would need to bump terra-common-lib to include the new kind of `JakartaTracing`, but of course doing this would require excluding various Liquibase deprecated methods in that lib from overriding a pinned version we depend on for core Liquibase functionality.

Tl;dr, we need to remove this or enter dependency hell that would involve changing our `Liquibase.update` to use a new non-deprecated method